### PR TITLE
DEV-8072 `new Worker` imports, using package paths

### PR DESCRIPTION
**Pull request by useafterfree**
## Fix

Imports were using bundled relative umd paths which may or may not exist in the compiled package
This new style uses new Worker which will bundle in the required file in the proper location for run time.

- [x] programmatically add in imports
- [x] `new Worker` style to leveraging vite bundler

## QA 
- [ ] verify all packages work with `npm run dev`
- [ ] deploy all packages (how do we do this?)
- [ ] verify deployed distributions ALSO work properly

[DEV-8072]

### CODE REVIEW

**Review by ericbarch — CHANGES_REQUESTED** [review]
Unfortunately looks like this didn't fix the issue and may have introduced additional issues.

Screenshot_20240429_132235

File: packages/data-sdk/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "preversion": "npm run verify",
     "postversion": "make",
-    "dev": "vite --port 5173",
+    "dev": "vite --port 5173 --host 0.0.0.0",
```
Comment (ericbarch) [inline packages/data-sdk/package.json:31]: Why opening up to all hosts? Could be an issue if working at coffee shops, random wifi, etc...

**Review by ericbarch — CHANGES_REQUESTED** [review]

File: packages/data-sdk/package.json (hunk 1 above)
Comment (useafterfree) [reply packages/data-sdk/package.json:31]: Working on VMs / remote hosts etc. requires this. I plan to remove this actually, and anyone using alternative dev paths can add :`npm run dev --host 0.0.0.0`

Comment (useafterfree) [reply packages/data-sdk/package.json:31]: many parts of this repo already have this:

image

**Review by ericbarch — APPROVED** [review]
LGTM :+1:

**Review by josorioromero — APPROVED** [review]

**Pull request merged**